internal/plans/objchange/objchange_test.go

```diff
@@ -954,7 +954,7 @@ func TestProposedNew(t *testing.T) {
 					}),
 					"c": cty.ObjectVal(map[string]cty.Value{
 						"bar": cty.StringVal("bosh"),
-						"baz": cty.NullVal(cty.List(cty.String)),
```

Comment:
Was this change part of the bug fix, or just correcting something else?. I see in the code that this change is part of the config object. Do we no longer represent config objects as `cty.NullVal(cty.List(cty.String))`?

Reply:
Thanks!